docs/installation.md

```diff
@@ -0,0 +1,136 @@
+| `datavolumes.cdi.kubevirt.io/v1beta1`                                   | `data-sources` operand (Kind `DataVolume` and `DataVolumeSource`)         |
+| `datasources.cdi.kubevirt.io/v1beta1`                                   | `data-sources` operand (Kind `DataSource`)                                |
+| `prometheusrules.monitoring.coreos.com`                                 | `metrics` operand (Kind `PrometheusRule`)                                 |
+| `template.openshift.io/v1`                                              | `common-templates` operand (Kind `Template`)                              |
+| `virtualmachineclusterinstancetypes.instancetypes.kubevirt.io/v1alpha2` | `common-instancetypes` operand (Kind `VirtualMachineClusterInstancetype`) |
+| `virtualmachineclusterpreferences.instancetypes.kubevirt.io/v1alpha2`   | `common-instancetypes` operand (Kind `VirtualMachineClusterPreference`)   |
+| `pipelines.tekton.dev`                                                  | `tekton-pipelines` operand (Kind `Pipeline`)                              |
+| `tasks.tekton.dev`                                                      | `tekton-tasks` operand (Kind `Task`)                                      |
+| `virtualMachine.kubevirt.io`                                            | `vm-controller` operand (Kind `VirtualMachine`)                           |
+
+### Kubernetes
+
+| Resource Type / CRD                                                     | Needed By                                                                 |
+| ------------------------------------------------------------------------| --------------------------------------------------------------------------|
+| `virtualmachineclusterinstancetypes.instancetypes.kubevirt.io/v1alpha2` | `common-instancetypes` operand (Kind `VirtualMachineClusterInstancetype`) |
+| `virtualmachineclusterpreferences.instancetypes.kubevirt.io/v1alpha2`   | `common-instancetypes` operand (Kind `VirtualMachineClusterPreference`)   |
+| `pipelines.tekton.dev`                                                  | `tekton-pipelines` operand (Kind `Pipeline`)                              |
+| `tasks.tekton.dev`                                                      | `tekton-tasks` operand (Kind `Task`)                                      |
+| `virtualMachine.kubevirt.io`                                            | `vm-controller` operand (Kind `VirtualMachine`)                           |
+
+## <a name="hco-installation"></a> Hyperconverged Cluster Operator (HCO) Installation
+
+This operator is typically deployed and controlled by
+[Hyperconverged Cluster Operator](https://github.com/kubevirt/hyperconverged-cluster-operator)
+automatically.
+
+## <a name="manually-installation"></a> Manually Installation
+
+This operator can be installed manually by applying the file `ssp-operator.yaml`
+from GitHub repository releases:
+```shell
+oc apply -f ssp-operator.yaml
```

Comment:
nit - I don't think this is helpful given the following commands.

Reply:
Yep, removed.